module.exports = {
  UNKNOWN_ERROR: 'Unknown error',
  UNAUTHORIZED_ERROR: 'User in not authorized',
  INVALID_CREDENTIALS: 'Invalid credentials',
  USER_ALREADY_EXISTS: 'User with this username already exists',
  CONTACT_ALREADY_EXISTS: 'User already added to contacts',
  CONTACT_NOT_EXIST: 'No such user in contacts',
  CONTACT_ADD_YOURSELF: 'Cannot add yourself',
  CONTACT_REMOVE_YOURSELF: 'Cannot remove yourself'
};
